packages/bdd-tests/step_definitions/shared-steps.ts

```diff
@@ -110,5 +117,42 @@ export const whenISendADispenseNotification = (when) => {
 export const whenISendADispenseNotificationForTheNolineItems = (when) => {
   when(/^I send a dispense notification for the (\d+) line items$/, async (medDispNo, table) => {
     resp = await helper.sendDispenseNotification(_site, medDispNo, table)
-  });
+  })
+}
+export const thePrescriptionIsMarkedAs = (then) => {
+  then(/^the prescription is marked as (.*)$/, (status) => {
+    //TODO
```

Comment:
We don't have the ability to check Status in spine on Internal Dev or Test, this is required. Not sure when it will be a priority to do

Reply:
Well,  that is why they are marked as todo, if they use it, it will still have to do because it is not being used. We can either comment out the whole step, or mark it as to do. But we cannot omit the vital test step. @MatthewPopat-NHS suggested putting todo. So what is your proposed solution?

Reply:
Personally, I'd remove the function altogether. If it's not providing any value at this point and may be misleading to others, it's a net negative.
Let's ask Matt what he thinks on Monday.

Reply:
Yeah, I see where you're coming from. I think the text that links to the function should also make it clear that it isn't implemented, in that case.